README.md

```diff
@@ -14,20 +13,21 @@ tests with the [Test Explorer UI](https://marketplace.visualstudio.com/items?ite
 
 * Install the extension
 * Configure Visual Studio Code to discover your tests
-  (see [Configuration section](#configuration) and documentation of a test framework of your choice
-  ([Unittest documentation](https://docs.python.org/3/library/unittest.html#module-unittest),
-   [Pytest](https://docs.pytest.org/en/latest/getting-started.html))
+  (see [Configuration section](#configuration) and documentation of a test framework of your choice:
+  * [Unittest documentation](https://docs.python.org/3/library/unittest.html#module-unittest)
+  * [Pytest documentation](https://docs.pytest.org/en/latest/getting-started.html)
+  * [Testplan documentation](https://testplan.readthedocs.io/en/latest/getting_started.html)
 * Open Test View sidebar
 * Run your tests using the ![Run](img/run-button.png) icon in the Test Explorer
 
 ## Features
 
 * Shows a Test Explorer in the Test view in VS Code's sidebar with all detected tests and suites and their state
 * Convenient error reporting during test discovery
-* Unittest and Pytest debugging
+* Unittest, Pytest and Testplan debugging
```

Comment:
Just curious, have you tested debugging?

Reply:
Great question, it is working now :)